rtos/Mail.h

```diff
@@ -300,16 +317,46 @@ class Mail : private mbed::NonCopyable<Mail<T, queue_sz> > {
      *         @a osErrorParameter A parameter is invalid or outside of a permitted range.
      *
      * @note You may call this function from ISR context if the millisec parameter is set to 0.
+     * @deprecated Replaced with try_get and try_get_for. In future get will be an untimed blocking call.
      */
-    osEvent get(Kernel::Clock::duration_u32 rel_time = Kernel::wait_for_u32_forever)
+    MBED_DEPRECATED_SINCE("mbed-os-6.0.0", "Replaced with try_get and try_get_for. In future get will be an untimed blocking call.")
+    osEvent get(uint32_t millisec = osWaitForever)
     {
-        osEvent evt = _queue.get(rel_time);
+        osEvent evt = _queue.get(millisec);
```

Comment:
Yes, but only when this deprecated `Mail::get` call is used, and this is method instantiated, because it's a template class.

So it means 2 deprecation messages rather than 1, but none if you're not using a deprecated method.

I think.